lib/getLocalChanges.js

```diff
@@ -30,26 +28,24 @@ function getLocalChanges (state, options, docsOrIds) {
     return Promise.reject(errors.NOT_AN_OBJECT)
   }
 
-  console.log('🙊  ' + docsOrIds)
-
-  return Promise.all([
-      localDB.info().then(toSeq),
-      remoteDB.info().then(toSeq)
-  ])
-  .then(function(args) {
-    if(args[0] === args[1]) {
-      return Promise.resolve([])
-    } else {
-      return Promise.resolve(args)
-    }
+  return genReplicationId(localDB, remoteDB, {})
+  .then(function (replicationId) {
+    return localDB.get(replicationId)
+  })
+  .then(function (localReplicationDoc) {
+    return localReplicationDoc.last_seq
+  })
+  .catch(function () {
+    return 0
```

Comment:
maybe let's add a comment here, explaining why we return 0? Something like this maybe ?

```
// if there was no replication yet, localDB.get will fail with 404. In that case, we want return all docs, as non has been synchronised yet
```